src/EdpGithub/Collection/RepositoryCollection.php

```diff
@@ -125,7 +126,7 @@ public function add($offset, $element)
         $this->elements[$offset] = $element;
     }
 
-    private function getPagination($response)
+    private function getPagination(Http\Response $response)
```

Comment:
@gianarb 

I believe it's safe to type-cast here, as there's only one place where it is used, which is

* https://github.com/EvanDotPro/EdpGithub/blob/8ac20f17414229e5e8964e9e343d1fcfe3669192/src/EdpGithub/Collection/RepositoryCollection.php#L90-L91